src/main/resources/assets/sophisticatedbackpacks/lang/tok.json

```
@@ -0,0 +1,259 @@
+  "item.sophisticatedbackpacks.battery_upgrade": "namako pi wawa ilo",
+  "item.sophisticatedbackpacks.battery_upgrade.tooltip": "lili poki li jo e wawa ilo",
+  "item.sophisticatedbackpacks.tank_upgrade": "namako telo",
+  "item.sophisticatedbackpacks.tank_upgrade.tooltip": "lili poki li jo e telo",
+  "item.sophisticatedbackpacks.pump_upgrade": "namako pi tawa telo",
+  "item.sophisticatedbackpacks.pump_upgrade.tooltip": "telo li ken tawa tan poki tawa ijo poka anu tan ijo poka tawa poki",
+  "item.sophisticatedbackpacks.xp_pump_upgrade": "namako tawa pi wawa laso",
+  "item.sophisticatedbackpacks.xp_pump_upgrade.tooltip": "wawa laso li ken tawa tan poki tawa sina anu tan sina tawa poki",
+  "item.sophisticatedbackpacks.advanced_pump_upgrade": "namako wawa pi tawa telo",
+  "item.sophisticatedbackpacks.advanced_pump_upgrade.tooltip": "telo li ken tawa tan poki tawa ijo poka anu tan ijo poka tawa poki\nijo jo kin li ken ijo poka\nwile la telo pi wile sina taso li ken tawa",
+  "item.sophisticatedbackpacks.tool_swapper_upgrade": "namako pi ante ilo",
+  "item.sophisticatedbackpacks.tool_swapper_upgrade.tooltip": "sina alasa pakala e ijo kepeken ilo ike la poki li pana e ilo pona",
+  "item.sophisticatedbackpacks.advanced_tool_swapper_upgrade": "namako wawa pi ante ilo",
+  "item.sophisticatedbackpacks.advanced_tool_swapper_upgrade.tooltip": "sina alasa pakala e ijo kepeken ilo ike la poki li pana e ilo pona\nnasin wile mute li lon\nilo pi pakala ala la sina ken kepeken nena %s",
+  "item.sophisticatedbackpacks.anvil_upgrade": "namako pi supa pona",
+  "item.sophisticatedbackpacks.anvil_upgrade.tooltip": "supa pona li lon poka poki",
+  "item.sophisticatedbackpacks.smithing_upgrade": "namako pi supa ilo",
+  "item.sophisticatedbackpacks.smithing_upgrade.tooltip": "supa ilo li lon poka poki",
+  "item.sophisticatedbackpacks.infinity_upgrade": "namako pi pini ala (lawa taso)",
+  "item.sophisticatedbackpacks.infinity_upgrade.tooltip": "ijo poki li pini ala\njan lawa taso li ken ante e ijo li ken jo e poki li ken pakala e poki",
+  "item.sophisticatedbackpacks.survival_infinity_upgrade": "namako pi pini ala (jan ale)",
+  "item.sophisticatedbackpacks.survival_infinity_upgrade.tooltip": "ijo poki li pini ala\njan ale taso li ken ante e ijo li ken jo e poki li ken pakala e poki",
+  "item.sophisticatedbackpacks.alchemy_upgrade": "namako pi telo wawa",
+  "item.sophisticatedbackpacks.alchemy_upgrade.tooltip": "poki li pana e telo wawa e moku wawa e telo soweli lon insa sina",
+  "item.sophisticatedbackpacks.advanced_alchemy_upgrade": "namako wawa pi telo wawa",
+  "item.sophisticatedbackpacks.advanced_alchemy_upgrade.tooltip": "poki li pana e telo wawa e moku wawa e telo soweli lon insa sina\nnasin wile mute li lon",
+  "item.sophisticatedbackpacks.chipped.botanist_workbench_upgrade": "namako pi supa pali kasi tan ilo Chipped",
+  "item.sophisticatedbackpacks.chipped.botanist_workbench_upgrade.tooltip": "supa pali kasi  li lon poka poki",
```

```suggestion
  "item.sophisticatedbackpacks.chipped.botanist_workbench_upgrade.tooltip": "supa pali kasi li lon poka poki",
```
(sitelen weka tu li lon insa nimi)